### ISSUE TITLE
Chimp is deleting all selenium sessions (not just its own)

### ISSUE THREAD

**Issue by bjarketrux** [xolvio/chimp#642]
When chimp is finishing a test run it will ask selenium for all sessions and then send a delete request for each of the sessions.

#####  Expected behaviour
Chimp should only delete the session(s) it have created itself (in this run, not other chimp processes).

#####  Actual behaviour
Chimp deletes all selenium processes, effectively killing all other UI tests.

##### Exact steps to reproduce / repository that demonstrates the problem
1. Setup a chimp test suite that uses a selenium server.
2. Start two different tests (e.g. from two different clients/computers)
3. Observe: When the first test finishes, the other test fails.

#####  Version & tools:

chimp@0.41.2
node@v4.8.4
OSX
Selenium server: Windows 7

#####  Console / Log Output
```
[chimp][hooks] Starting AfterFeatures
[chimp][hooks] Ending browser session
[chimp][session-manager] requesting sessions from http://seleniumserver:4444/wd/hub/sessions
[chimp][session-manager] received data {...}
[chimp][session-manager] deleting wd session 6359067f-d110-4344-b416-a020ec7d22f6
[chimp][session-manager] deleting wd session 5e26b203-88f7-4f65-a720-00d827172616
[chimp][session-manager] received data {...}
[chimp][hooks] Ended browser sessions
[chimp][hooks] Finished AfterFeatures
```

### Note
In `session-manager.js` the comment for `killCurrentSession` is `Kills the 1st session found running on selenium server` but the code indicates that it loops through all sessions: `sessions.forEach(function (session) {`
It also seems that `noSessionReuse` could prevent this, but it is not working.

------- 
*Join our Slack [xolv.io/community](http://xolv.io/community) #chimp channel, where you can find help and help others.*

**Comment by bjarketrux** [comment]
Can be partially fixed by #643. Then just use `noSessionReuse` option to leave all sessions. But the better solution is probably to only delete the one session that chimp created.

**Comment by samhatoum** [comment]
This issue is being tracked in Chimpy 
https://github.com/TheBrainFamily/chimpy/issues
